.drone.yml

```diff
@@ -1709,20 +1769,30 @@ steps:
   - which curl || apk add --no-cache curl
   - |-
     cd "$WORKSPACE_DIR/go/artifacts"
-    for file in $(find . -type f ! -iname '*.sha256'); do
+    for file in $(find . -type f ! -iname '*.sha256' ! -iname '*-unsigned.zip*'); do
```

Comment:
This pipeline (lines 1670-1812) backported semi-manually, as it does not exist in v9/master, and dronegen is gone in v8.